cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -71,153 +92,158 @@ env:
   value: YOUR_AWS_REGION
 ```
 
-## Deployment Specification
-Auto-Discovery Setup is always preferred option to avoid multiple, potentially different configuration for min/max values. If you want to adjust minimum and maximum size of the group, please adjust size on ASG directly, CA will fetch latest change when talking to ASG.
+## Auto-Discovery Setup
```

Comment:
Do we need to make this change?  Probably change to following makes more sense? They should belong to one header 2

Deployment Specification
- Auto-Discovery
- Manual Configuration
  - One ASG
  - Multiple ASG
- Master Node Setup